### PR TITLE
run tests on travis

### PR DESCRIPTION
**Pull request by crew102**
Pretty self-explanatory. This runs tests on Travis but will still skip them on CRAN. I tested it out by adding a failing test and it seems to work.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,6 @@ r_github_packages:
 after_success:
   - Rscript -e 'covr::codecov()'
 
-r_check_args: "--as-cran"
```
Comment (sckott) [inline .travis.yml]: why remove this?

Comment (sckott) [reply .travis.yml]: A few things. 

I do have that env var set on travis, but env vars are NOT available on pull requests. Tests are using cached "fixtures" via the `vcr` package. So no real HTTP requests are done in the test suite. See the cached files in tests/fixtures/

I don't think `--as-cran` should affect whether tests are run or not.

Comment (crew102) [reply .travis.yml]: Oh, I didn't realize you had set the API key as an env var in Travis. That's why the tests were passing. My solution was to just set a phony key as an env var when the tests are run on travis, with the idea being that as long as there was a non-empty value for MICROSOFT_ACADEMIC_KEY then `check_key()` wouldn't error and the tests would pass (b/c, as you mentioned, no actual requests would be sent out on Travis b/c of the use of vcr, so it wouldn't matter what the value of the key was).

Comment (sckott) [reply .travis.yml]: Tests ran fine on the PR travis build, and if I unset the env var locally, the test fail. 

So I think you're right that we do need to set a string for the key, any string, for tests to work. 

can you reopen the PR and just remove the as-cran change?

Comment (crew102) [reply .travis.yml]: Opps, didn't realize that a force push on this branch was going to disallow the reopening of the PR. I'll just link to this PR from a fresh one.

**Pull request closed without merging**